force-app/main/default/classes/SOQL.cls

```diff
@@ -248,7 +311,11 @@ public with sharing class SOQL {
      * @return The Account Name of the Contact.
      */
     public static String getAccountNameForContact(String contactId) {
-        return null; // Replace null with the variable you used to store the result
+
+        
+        Contact con =  [SELECT Id, Name , AccountId FROM Contact WHERE Id = :contactId];
+        Account acc = [SELECT Name FROM Account WHERE Id = :con.AccountId];
+        return acc.Name;
```

Comment:
You should be able to do this with just 1 query. How would we access lookup fields in a query?